apps/meteor/app/ui-utils/lib/MessageTypes.ts

```diff
@@ -9,7 +9,7 @@ export type MessageType = {
 	/* deprecated */
 	template?: (message: IMessage) => unknown;
 	message: TranslationKey;
-	data?: (message: IMessage) => Record<string, string>;
+	data?: (message: IMessage) => Record<string, string> | undefined;
```

Comment:
I don't think the data() method is equipped to deal with an undefined value. I'd recommend instead returning an empty string corresponding to the parameter

Reply:
Fixed.